gourmet/__init__.py

```diff
@@ -22,3 +22,10 @@
 # if args.thread_debug:
 #     import threading
 #     thread_debug()
+
+# specify the required GI namespaces version here.
+# 'require_version' only needs to be called once to ensures the namespace gets loaded with the given version
+from gi import require_version
+require_version("Gdk", "3.0")
+require_version('Gst', '1.0')
```

Comment:
I think you accidentally removed Pango from this list instead of Gst. Pango should be kept here, as far as I know - it's used in several places.

Reply:
Thanks for spotting this.